<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grade Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Final Grade:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="FinalGrade">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0.000000%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="SchemaA">
        <property name="text">
         <string>Schema A</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SchemaB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Schema B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>PIC 10B</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>HW1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW1">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>HW2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW2">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>HW3</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW3">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>HW4</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW4">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>HW5</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW5">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>HW6</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW6">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>HW7</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BHW7">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Midterm 1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BMT1">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Midterm 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BMT2">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Final Exam</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="PIC10BFE">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque sed porttitor urna. Sed lobortis ligula a odio tincidunt tristique. Aliquam sed felis a neque blandit volutpat. Sed nec ante nisl. Donec a est vitae erat commodo aliquet. Etiam pharetra, ante non venenatis fringilla, tellus arcu rutrum enim, nec convallis sem sapien ut lorem. In quis lorem vitae erat varius congue. Vestibulum ultrices sagittis tristique. Sed eget blandit tellus, ut sollicitudin leo. Pellentesque et enim elementum, varius urna id, scelerisque mauris. Quisque imperdiet felis at nulla accumsan efficitur. Vestibulum leo dolor, commodo accumsan porttitor quis, pulvinar vitae erat.

Quisque feugiat, nunc sed vehicula aliquam, diam purus vulputate ipsum, id sodales magna metus nec augue. Curabitur ultricies semper sapien eget dignissim. Vestibulum facilisis quis nisl vitae ultrices. Nullam ultrices leo id euismod venenatis. Maecenas est quam, gravida non ipsum et, scelerisque luctus neque. Phasellus pellentesque erat non vehicula pellentesque. Pellentesque ac turpis leo. Duis diam lacus, consequat vitae vehicula ac, finibus ac erat. Nullam at faucibus lectus.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>PIC 10C</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Project 1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CHW1">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Project 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CHW2">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Project 3</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CHW3">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Midterm</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CMT">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Final Exam</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CFE">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Final Project</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="PIC10CFP">
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque sed porttitor urna. Sed lobortis ligula a odio tincidunt tristique. Aliquam sed felis a neque blandit volutpat. Sed nec ante nisl. Donec a est vitae erat commodo aliquet. Etiam pharetra, ante non venenatis fringilla, tellus arcu rutrum enim, nec convallis sem sapien ut lorem. In quis lorem vitae erat varius congue. Vestibulum ultrices sagittis tristique. Sed eget blandit tellus, ut sollicitudin leo. Pellentesque et enim elementum, varius urna id, scelerisque mauris. Quisque imperdiet felis at nulla accumsan efficitur. Vestibulum leo dolor, commodo accumsan porttitor quis, pulvinar vitae erat.

Quisque feugiat, nunc sed vehicula aliquam, diam purus vulputate ipsum, id sodales magna metus nec augue. Curabitur ultricies semper sapien eget dignissim. Vestibulum facilisis quis nisl vitae ultrices. Nullam ultrices leo id euismod venenatis. Maecenas est quam, gravida non ipsum et, scelerisque luctus neque. Phasellus pellentesque erat non vehicula pellentesque. Pellentesque ac turpis leo. Duis diam lacus, consequat vitae vehicula ac, finibus ac erat. Nullam at faucibus lectus.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>updatePIC10B()</slot>
 </slots>
</ui>
